server-common/src/main/java/org/apache/gravitino/server/authorization/MetadataAuthzHelper.java

```diff
@@ -69,10 +74,6 @@ public static NameIdentifier[] filterByPrivilege(
       Entity.EntityType entityType,
       String privilege,
       NameIdentifier[] metadataList) {
-    if (!enableAuthorization()) {
```

Comment:
Why do u remove this?

Reply:
There is duplicate logic in doFilter.

Reply:
This method doesn't call the method `doFilter`.